JellyfishClient/src/main/java/org/membraneframework/rtc/utils/EndpointConnectionUtils.kt

```diff
@@ -0,0 +1,18 @@
+package org.membraneframework.rtc.utils
+
+import org.webrtc.*
```

Comment:
Maybe lets import only necessary clasess?

Reply:
Good idea, but as it is leftover from `membrane-webrtc-android`, let's do it in separate PR